File: Beecrowd/1796 Economia Brasileira/main.c
#include <stdio.h>

int main(){
    int q, sas, cont = 0, cont2 = 0;

    scanf("%d", &q);

    while(q--){
        scanf("%d", &sas);

        if(sas == 1)
            cont++;
        else
            cont2++;
    }

    if(cont >= cont2)
        printf("N\n");
    else
        printf("Y\n");

    return 0; 
}

File: Beecrowd/1796 Economia Brasileira/main.py
cont, cont2 = 0, 0

q = int(input())

entrada = [int(_) for _ in input().split()]
satisfacao = entrada[0:q]

for sas in satisfacao:
    if sas == 1:
        cont += 1
    else:
        cont2 += 1

if cont >= cont2:
    print("N")
else:
    print("Y")